packages/terra-embedded-content-consumer/src/terra-dev-site/test/embedded-content-consumer/providers/CustomEventProvider.test.jsx

```diff
@@ -1,36 +0,0 @@
-import React from 'react';
```

Comment:
Renamed to src/terra-dev-site/provider/embedded-content-consumer/providers/CustomEventProvider.provider.jsx